docs/root/configuration/overview/_include/xds_api/oauth-sds-example.yaml

```diff
@@ -0,0 +1,137 @@
+                redirect_uri: http://localhost:8000/callback
+                redirect_path_matcher:
+                  path:
+                    exact: "/callback"
+                signout_path:
+                  path:
+                    exact: "/signout"
+                forward_bearer_token: true
+                credentials:
+                  client_id: "my-client-id"
+                  token_secret:
+                    name: token
+                    sds_config:
+                      resource_api_version: V3
+                      api_config_source:
+                        api_type: GRPC
+                        transport_api_version: V3
+                        grpc_services:
+                        - envoy_grpc:
+                            cluster_name: sds_server_uds
+                  hmac_secret:
+                    name: hmac
+                    sds_config:
+                      resource_api_version: V3
+                      api_config_source:
+                        api_type: GRPC
+                        transport_api_version: V3
+                        grpc_services:
+                        - envoy_grpc:
+                            cluster_name: sds_server_uds
+                auth_scopes:
+                - openid
+          - name: envoy.filters.http.jwt_authn
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.jwt_authn.v3.JwtAuthentication
+              providers:
+                provider1:
+                  remote_jwks:
+                    http_uri:
+                      uri: https://myoauthprovider.net/oauth2/default/v1/keys
+                      cluster: oauth
+                      timeout: 5s
+                    cache_duration: 600s
+              rules:
+              - match:
+                  prefix: "/"
+                requires:
+                  provider_name: provider1
+          - name: envoy.filters.http.router
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
+  clusters:
+  - name: sds_server_uds
+    typed_extension_protocol_options:
+      envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+        "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+        explicit_http_config:
+          http2_protocol_options: {}
+    load_assignment:
+      cluster_name: sds_server_uds
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              pipe:
+                path: /tmp/uds_path
+  - name: upstream-service
+    connect_timeout: 2s
+    type: STRICT_DNS
+    lb_policy: ROUND_ROBIN
+    load_assignment:
+      cluster_name: upstream-service
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: 127.0.0.1
+                port_value: 8000
+  - name: oauth
+    connect_timeout: 2s
```

Comment:
i happen to know that this is not required so we can remove (and elsewhere if present)

i dont know the rest of the configs so well - so if you can spot anything at all that is not strictly necessary then please remove - this will just make it easier for end users to know exactly what is required for each setup

Reply:
thanks @phlax, I have kept the configs now minimal, but for this oauth-sds-example - if you see it being complex more than required to explain xds_api, we can omit this and use a different example